deisctl/units/deis-controller.service

```diff
@@ -6,8 +6,10 @@ After=deis-store-volume.service
 [Service]
 EnvironmentFile=/etc/environment
 TimeoutStartSec=20m
+ExecStartPre=-/usr/bin/etcdctl mkdir /deis/scheduler >/dev/null 2>&1
```

Comment:
What if the etcd directory already exists? I think this is how we meant to ignore that error.
```console
core@deis-01 ~ $ /usr/bin/etcdctl mkdir /deis/foobar
Error:  105: Key already exists (/deis/foobar) [67559]
core@deis-01 ~ $ echo $?
4
```